docs/faq/subgraphAPI.md

```diff
@@ -0,0 +1,99 @@
+## How to Integrate Backend Libraries to MXNet with the Subgraph API
+
+The subgraph API has been proposed and implemented as the default mechanism for integrating backend libraries to MXNet. The subgraph API is a very flexible interface. Although it was proposed as an integration mechanism, it has been used as a tool for manipulating NNVM graphs for graph-level optimizations, such as operator fusion.
+
+The subgraph API works as the following steps:
+
+* Search for particular patterns in a graph.
+* Group the operators/nodes with particular patterns into a subgraph and shrink the subgraph into a single node.
+* Put the subgraph node back to the original graph to replace the subgraph.
+
+The figure below illustrates the subgraph mechanism.
+
+![](./subgraph.png)
+
+The subgraph API allows the backend developers to customize the subgraph mechanism in two places:
+
+* Subgraph searching: define a subgraph selector to search for particular patterns in a computation graph.
+* Subgraph node creation: attach an operator to run the computation in the subgraph. We can potentially manipulate the subgraph here.
+
+
+Below I will illustrate the subgraph API with a very simple task, as shown in the figure above. That is, replacing convolution and batch_norm with the conv_bn.
```

Comment:
Let's not use "I" in a FAQ or tutorial. Switch your content throughout.
Also refrain from above and below.
Try:
"The following is a demonstration of how the Subgraph API can be applied to a simple task. Refer to the previous figure for an overview of the process."

So if we were in tablet swiping mode, previous and next make sense. Same for desktop. But above and below don't make sense.